misc/swagger-ballerina/modules/ballerina-to-swagger/src/main/java/org/ballerinalang/ballerina/swagger/convertor/service/model/Developer.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
@sanjeewa-malalgoda better to make it 2018

Reply:
This was written in 2017 and im moving it to different location. Do we changing licence headers in such situations?